File: No_70.cs
namespace LeetCode
{
    public class No_70
    {
        public int ClimbStairs(int n)
        {
            if (n < 4) return n;
            int ans = 0, a = 2, b = 3;

            for (int i = 4; i <= n; i++)
            {
                ans = a + b;
                a = b;
                b = ans;
            }
            return ans;
        }
    }
}

File: No_70.ts
namespace Leetcode{
    export class No_70 {
       climbStairs(n: number): number {

        if(n < 4)
            return n;
        let ans = 0;
        let a = 2;
        let b = 3;

        for(let i = 4; i <= n ; i++){
            ans = a + b;
            a = b;
            b = ans;
        }

        return ans;
};
    }
}